components/InviteAdminModal/InviteAdminModal.tsx

```diff
@@ -0,0 +1,283 @@
+                    />
+                  </div>
+
+                  {i != 0 && (
+                    <IconButton
+                      variant="contained"
+                      onClick={() => removeRow(i)}
+                    >
+                      <Icon type={"inviteTrash"} />
+                    </IconButton>
+                  )}
+                  {i == 0 && <div className={styles["placeholder"]}></div>}
+                </div>
+              ))}
+              <div className={styles["row"]} />
+              {invites.length != 5 && (
+                <IconButton
+                  variant="contained"
+                  onClick={addRow}
+                  // style={{ backgroundColor: 'transparent' }}
+                  className={styles["add"]}
+                >
+                  <Icon type={"inviteAdd"} className={styles["addIcon"]} />
+                  New Invitation
+                </IconButton>
+              )}
+            </div>
+          </div>
+          <div className={styles["actions"]}>
+            <DialogActions>
+              <ThemeProvider theme={theme}>
+                <Button
+                  style={{ textTransform: "none" }}
+                  className={styles["buttons"]}
+                  variant="outlined"
+                  onClick={handleClose}
+                >
+                  Cancel
+                </Button>
+                <Button
+                  type="submit"
+                  className={styles["buttons"]}
+                  variant="contained"
+                >
+                  Send Invites
+                </Button>
+              </ThemeProvider>
+            </DialogActions>
+          </div>
+        </form>
+      </div>
+    );
+  };
+
+  const renderSuccessPage = () => {
+    return (
+      <div className={styles["success"]}>
+        <div className={styles["content"]}>
+          <h1 className={styles["header"]}>Success!</h1>
+          <p className={styles["message"]}>
+            All emails have been sent to their respective addresses
+          </p>
+          <div className={styles["actions"]}>
+            <ThemeProvider theme={theme}>
+              <Button
+                style={{ textTransform: "none" }}
+                className={styles["buttons"]}
+                variant="outlined"
+                onClick={handleClose}
+              >
+                Close
+              </Button>
+              <Button
+                type="submit"
+                className={styles["invite-more"]}
+                variant="contained"
+                onClick={() => setSent(false)}
+              >
+                Invite More
+              </Button>
+            </ThemeProvider>
+          </div>
+        </div>
+      </div>
+    );
+  };
+
+  return (
+    <div>
+      <ThemeProvider theme={theme}>
+        <Button
+          variant="contained"
+          style={{ textTransform: "none" }}
+          onClick={handleClickOpen}
+        >
+          Invite Admin
```

Comment:
I think this button should be in the index.tsx file in the admins folder, all components we see here should just correspond to what we see on the modal